eole/scorers/comet.py

```diff
@@ -0,0 +1,46 @@
+from .scorer import Scorer
+from eole.scorers import register_scorer
+from comet import download_model, load_from_checkpoint
+
+
+@register_scorer(metric=["COMET", "COMET-KIWI", "COMETINHO"])
+class CometScorer(Scorer):
+    """Comet scorer class."""
+
+    def __init__(self, config, model_name):
+        """Download if needed and load Comet model."""
+        super().__init__(config, model_name)
+        self.model_name = model_name
+        if self.model_name == "COMET":
+            comet_model_name = "Unbabel/wmt22-comet-da"
+        elif self.model_name == "COMETINHO":
+            comet_model_name = "Unbabel/eamt22-cometinho-da"
+        elif self.model_name == "COMET-KIWI":
+            comet_model_name = "Unbabel/wmt22-cometkiwi-da"
+        try:
+            comet_model_path = download_model(comet_model_name)
+        except KeyError:
+            raise PermissionError(
+                "Use your hugginface token to download comet-kiwi:\n"
+                "huggingface-cli login --token hf_token"
+            )
+        self.comet_model = load_from_checkpoint(comet_model_path)
+
+    def compute_score(self, preds, texts_refs, texts_srcs):
+        data = []
+        if self.model_name == "COMET-KIWI":
+            for _src, _hyp in zip(texts_srcs, preds):
+                current_segment = {"src": _src, "mt": _hyp}
+                data.append(current_segment)
+        else:
+            for _src, _tgt, _hyp in zip(texts_srcs, texts_refs, preds):
+                current_segment = {"src": _src, "mt": _hyp, "ref": _tgt}
+                data.append(current_segment)
+        if len(preds) > 0:
+            score = self.comet_model.predict(
+                data, batch_size=64, gpus=0, num_workers=0, progress_bar=True
```

Comment:
`batch_size` is a bit hidden here. Do we expect to need to change this at some point?
Might not be necessary to have a dedicated flag for this, but maybe set this explicitly in the __init__ so that it's a bit more explicit.